wemake_python_styleguide/visitors/ast/complexity/function.py

```diff
@@ -167,3 +182,18 @@ def visit_any_function(self, node: AnyFunctionDef) -> None:
         self._counter.check_arguments_count(node)
         self._counter.check_function_complexity(node)
         self.generic_visit(node)
+
+    def visit_Lambda(self, node: ast.Lambda) -> None:
+        """
+        Checks lambda function's internal complexity.
+
+        Raises:
```

Comment:
Please, check what errors can be raised by this visitor. I guess it is only a single one: arguments count.